Guess the Countries/README.md

```diff
@@ -0,0 +1,18 @@
+Package/Script Name
+# Guess the Country
+
+
+<p> Guess the name of the countries on the world map and score a point with every correct guess.</p>
+
+
+Output
+<img src='blank.png' width='350'>
```

Comment:
Images not rendered:

![image](https://user-images.githubusercontent.com/43691873/113384956-c5f4b380-93a4-11eb-8a23-3e4b79f94553.png)